functional_tests/lightning/.gitignore

```diff
@@ -0,0 +1 @@
+None/
```

Comment:
❓

Reply:
I inherited the test, havent tracked down where it came from.. removed gitignore to remind me to do so